client/state/jetpack-connect/selectors.js

```diff
@@ -11,6 +11,7 @@ import { get, includes } from 'lodash';
 import { AUTH_ATTEMPS_TTL } from './constants';
 import { getSiteByUrl } from 'state/sites/selectors';
 import { isStale } from './utils';
+import { getInitialQueryArguments } from 'state/selectors/get-initial-query-arguments';
```

Comment:
You could also directly import from the global selectors directory here - `state/selectors`.